src/app/appmarket/appdetails/appdetails.component.html

```diff
@@ -63,7 +63,7 @@ <h2 style="margin-bottom: 4px;">{{app?.name}}</h2>
 				</div>
 				<div *ngIf="subscribed" class="btn-group pull-right" >
 					<div class="btn no-padding" pTooltip="{{'APPLICATIONS.TOOLTIP_MESSAGE_DEPLOY' | translate}}" tooltipPosition="bottom" [showDelay]="50" [tooltipDisabled]="defaultTooltipDisabled">
-						<button *ngIf="isDeploymentAllowed()" class="btn btn-primary first-button-substitute" [disabled]="!isApplicationEnabledInDomain()" (click)="appInstallModal.show()">{{'APPLICATIONS.DEPLOY_BUTTON' | translate}}</button>
+						<button *ngIf="isSubscriptionAllowed()" class="btn btn-primary first-button-substitute" [disabled]="!isApplicationEnabledInDomain()" (click)="appInstallModal.show()">{{'APPLICATIONS.DEPLOY_BUTTON' | translate}}</button>
```

Comment:
So this was actually bug in the logic?

Reply:
Both function (`isDeploymentAllowed` and `isSubscriptionAllowed` had the same code (probably some changes throughout the time)